GitCommands/Settings/AppSettings.cs

```diff
@@ -33,6 +33,7 @@ public static class AppSettings
         public static DistributedSettings SettingsContainer { get; private set; }
 
         private static readonly SettingsPath DetailedSettingsPath = new AppSettingsPath("Detailed");
+        private static readonly SettingsPath ConfirmationsSettingsPath = new AppSettingsPath("Confirmations");
```

Comment:
Class members of the same category should be sorted alphabetically (when added - although there are lots of deviations from this style rule in the existing code).
```suggestion
        private static readonly SettingsPath ConfirmationsSettingsPath = new AppSettingsPath("Confirmations");
        private static readonly SettingsPath DetailedSettingsPath = new AppSettingsPath("Detailed");
```